OscQueryLibrary/Models/JsonSourceGenerationContext.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Text.Json.Serialization;
+
+namespace OscQueryLibrary.Models;
+
+[JsonSourceGenerationOptions(WriteIndented = true)]
+[JsonSerializable(typeof(HostInfo))]
+[JsonSerializable(typeof(RootNode))]
+[JsonSerializable(typeof(AvatarContents))]
+[JsonSerializable(typeof(OscParameterNode))]
+[JsonSerializable(typeof(OscParameterNodeEnd<string>))]
+internal partial class JsonSourceGenerationContext : JsonSerializerContext

```

Comment:
You can shorten this if you'd like, as it's quite a mouthful. Just went with a default expressive name!